src/search/components/dateTime.component.tsx

```diff
@@ -295,45 +323,23 @@ const DateTimeSearch = (props: DateTimeSearchProps): React.ReactElement => {
               onClose={() => setPopupOpen(false)}
               onError={(error) => setDatePickerFromDateError(!!error)}
               views={['year', 'month', 'day', 'hours', 'minutes']}
-              OpenPickerButtonProps={{
-                size: 'small',
-                'aria-label': 'from, date-time picker',
+              slots={{
+                textField: CustomTextField,
               }}
```

Comment:
Aren't we missing specifying the custom `day`?

Reply:
Yes, this commit (e1f879d) moved the custom day to slots, but missed one in the "from" picker. I'll add it back.